### ISSUE TITLE
Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by DoM1niC** [Johni0702/mumble-web#121]
Whats wrong with this?

```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at ./node_modules/webworkify/index.js.module.exports (index.js:9)
    at ReusePool._factory (decoder-stream.js:36)
    at ReusePool../node_modules/reuse-pool/index.js.ReusePool.get (index.js:23)
    at Object../node_modules/mumble-client-codecs-browser/lib/decoder-stream.js (decoder-stream.js:39)
    at __webpack_require__ (bootstrap:19)
    at Object../node_modules/mumble-client-codecs-browser/lib/index.js (index.js:13)
    at __webpack_require__ (bootstrap:19)
    at Object../node_modules/mumble-client-codecs-browser/index.js (index.js:1)
    at __webpack_require__ (bootstrap:19)
```

**Comment by Johni0702** [comment]
Duplicate of https://github.com/Johni0702/mumble-web/issues/92
TLDR: You must not use the `root` user to build mumble-web. NPM will behave differently (and not fully build everything) when run as root.
